src/charm.py

```diff
@@ -1174,7 +1451,7 @@ def _reconciliation(self, _event):
             logger.info("Reconciliation process finished successfully.")
             self.unit.status = ops.model.ActiveStatus()
         except exceptions.WordPressStatusException as status_exception:
-            logger.info(f"Reconciliation process terminated early, reason: {status_exception}")
+            logger.info("Reconciliation process terminated early, reason: %s", status_exception)
             self.unit.status = status_exception.status
 
 
```

Comment:
This file is getting quite long, at what point do you think this should be broken up into multiple files?

Reply:
The upper half of this file is old code that's not needed anymore, I will remove them in a future PR so it will cut the file length in half.